src/java.base/share/classes/java/text/Collator.java

```diff
@@ -72,14 +72,15 @@
  * The following example shows how to compare two strings using
  * the {@code Collator} for the default locale.
  * <blockquote>
- * <pre>{@code
+ * {@snippet lang=java :
  * // Compare two strings in the default locale
  * Collator myCollator = Collator.getInstance();
- * if( myCollator.compare("abc", "ABC") < 0 )
+ * if (myCollator.compare("abc", "ABC") < 0) {
  *     System.out.println("abc is less than ABC");
- * else
+ * } else {
  *     System.out.println("abc is greater than or equal to ABC");
- * }</pre>
+ * }
+ * }
```

Comment:
I don't think you need a second '}' here.

Reply:
I see that now!  I didn't look up far enough.  Thanks!

Reply:
No worries, thank you for reviewing!